apparmor.d/profiles-a-f/briar-desktop

```diff
@@ -0,0 +1,97 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2024 Roman Beslik <me@beroal.in.ua>
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = @{bin}/briar-desktop
+profile briar-desktop @{exec_path} {
+  include <abstractions/audio-client>
+  include <abstractions/base>
+  include <abstractions/dconf-write>
+  include <abstractions/fontconfig-cache-read>
+  include <abstractions/gnome-strict>
+  include <abstractions/graphics>
+  include <abstractions/nameservice-strict>
+
+  network inet dgram,
+  network inet stream,
+  network inet6 dgram,
+  network inet6 stream,
+
+  @{lib}/jvm/java*/lib/jspawnhelper Cx -> jspawnhelper,
```

Comment:
Move this rule with the other '@{lib}/...` rules

Reply:
No, `change_profile` is a rule by itself, while `@{lib}/.. Cx` is a file rule even if it is about changing profile.